geonode/templates/metadata_base.html

```diff
@@ -1,234 +1,237 @@
-{% extends "geonode_base.html" %}

```

Comment:
You completely changed the format of this file, it is impossible to understand what's changed!